src/components/skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,8 @@
+import clsx from 'clsx'
+import classes from './styles.module.css'
+
+type SkeletonProps = React.HTMLAttributes<HTMLDivElement>
+
+export default function Skeleton({className, ...props}: SkeletonProps) {
```

Comment:
needs to be named export

Reply:
you also need to reexport from the index file or else it won't be accessible

Reply:
Done